### PR TITLE
docs(pre-commit): fix pre-commit YAML style

### PR DESCRIPTION
**Pull request by MicaelJarniac**
Docs only, changed `.pre-commit-config.yaml` style to match pre-commit's own examples and files.

Changed pre-commit config examples from
```yaml
repos:
  - repo: https://github.com/python-poetry/poetry
    hooks:
      - id: poetry-check
```
to
```yaml
repos:
-   repo: https://github.com/python-poetry/poetry
    hooks:
    -   id: poetry-check
```
in order to match pre-commit's own style:
https://pre-commit.com/#2-add-a-pre-commit-configuration
https://github.com/pre-commit/pre-commit/blob/main/.pre-commit-config.yaml

### CODE REVIEW

File: docs/pre-commit-hooks.md [hunk 1]
```diff
@@ -109,16 +109,16 @@ repos:
 `pre-commit autoupdate` updates the `rev` for each repository defined in your `.pre-commit-config.yaml`
 to the latest available tag in the default branch.
 
-Poetry follows a branching strategy, where the default branch is the active development branch
-and fixes gets back ported to stable branches. New tags are assigned in these stable branches.
+Poetry follows a branching strategy where the default branch is the active development branch
+and, fixes get backported to stable branches. New tags are assigned in these stable branches.
```
Comment (MicaelJarniac) [inline docs/pre-commit-hooks.md:113]:
```suggestion
Poetry follows a branching strategy where the default branch is the active development branch,
and fixes get backported to stable branches. New tags are assigned in these stable branches.
```

**Comment by dimbleby** [comment]
I see reason to go along with pre-commit's oddball yaml style!

**Comment by Secrus** [comment]
I agree with dimbleby on that. Grammar changes seem ok to me.

**Comment by MicaelJarniac** [comment]
> I see no reason to go along with pre-commit's oddball yaml style!

One possible reason, and what actually happened to me, is that when someone sets up pre-commit, they'll likely copy the example and the style from pre-commit's docs, and when they want to add Poetry's hooks, they'll copy it from Poetry's docs and paste it there, and notice the style doesn't match, so they'll have to manually fix it.

pre-commit's YAML style might be odd, but there's a good chance people's `.pre-commit-config.yaml` will be following that style, since it's what pre-commit itself uses.

But I'll be opening a separate PR with just the grammar changes.

**Comment by MicaelJarniac** [comment]
Modified this PR to be just about the YAML style, and opened a separate one for the grammar changes. Feel free to close this one if the YAML changes are totally undesirable.

**Review by Secrus — APPROVED** [review]

**Pull request merged**